### PR TITLE
fix: remove feature gate for action description

### PR DESCRIPTION
**Pull request by raquelmsmith**
## Problem

The action description field should not be feature gated, but it is.

## Changes

Removes the feature gating on that field. Also removes the event definition description field because it doesn't work.

👉 _Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review._

## Does this work well for both Cloud and self-hosted?

Yep

## How did you test this code?

Locally. Snapshots will prob update.

### CODE REVIEW

File: frontend/__snapshots__/scenes-other-settings--settings-user--light.png
Comment (zlwaterfield) [inline frontend/__snapshots__/scenes-other-settings--settings-user--light.png:1]: Weird that these Max images broke.

File: frontend/src/scenes/data-management/definition/DefinitionView.tsx [hunk 1]
```diff
@@ -133,20 +132,6 @@ export function DefinitionView(props: DefinitionLogicProps = {}): JSX.Element {
             />
 
             <div className="space-y-2">
-                <EditableField
```
Comment (zlwaterfield) [inline frontend/src/scenes/data-management/definition/DefinitionView.tsx:136]: Don't we still want this without the paywall?

**Review by zlwaterfield — COMMENTED** [review]
Should we move `INGESTION_TAXONOMY` in the billing plan features as well?

**Review by zlwaterfield — APPROVED** [review]

File: frontend/src/scenes/data-management/definition/DefinitionView.tsx (hunk 1 above)
Comment (zlwaterfield) [inline frontend/src/scenes/data-management/definition/DefinitionView.tsx:136]: Will this mean that the description won't show anymore? 

Screenshot 2024-06-04 at 2 41 07 PM

Screenshot 2024-06-04 at 2 41 13 PM

Comment (raquelmsmith) [reply frontend/src/scenes/data-management/definition/DefinitionView.tsx:136]: How were you able to edit that?? I couldn't edit it haha

Comment (zlwaterfield) [reply frontend/src/scenes/data-management/definition/DefinitionView.tsx:136]: "Edit" in the top right, and then a textarea appears. I can't edit the field inline.

**Comment by raquelmsmith** [comment]
> Should we move INGESTION_TAXONOMY in the billing plan features as well?

No, it's still used (debatable whether or not it should be) but it was being used incorrectly here - this should have been gated on the TEAM_COLLABORATION feature that was removed in https://github.com/PostHog/billing/pull/596 and https://github.com/PostHog/posthog/pull/22170

File: frontend/src/scenes/data-management/definition/DefinitionView.tsx (hunk 1 above)
Comment (raquelmsmith) [reply frontend/src/scenes/data-management/definition/DefinitionView.tsx:136]: OH. You have to have the INGESTION_TAXONOMY feature to see the edit button. 

No edit button without INGESTION_TAXONOMY:

Arc 2024-06-04 11 47 41

Will update, thanks!

**Pull request merged**